src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -203,9 +664,95 @@ internal void ClearSelection()
             WorkspaceViewModel.DynamoViewModel.ExecuteCommand(selectNothing);
         }
 
-        private bool CanChangeFontSize(object obj)
+        /// <summary>
+        /// Collapse all nodes in this group
+        /// by settings its IsCollapsed property
+        /// to true.
+        /// Only the Parent group should handle collapsing
+        /// the connector that needs it.
+        /// </summary>
+        /// <param name="collapseConnectors"></param>
+        private void CollapseGroupNodes(bool collapseConnectors)
```

Comment:
Considering we are collapsing all content under group, can we rename this to `CollapseGroupContent` and also the function below from `ShowGroupNodes` to `ShowGroupContent`?

Reply:
yea the makes more sense. The only reason i stuck with Nodes was that the "content" property is called `Nodes` which might also be a bad name as it can contain both `NoteModels`, `NodeModels` and now `AnnotationModels`.

Will use content for these 👍